### ISSUE TITLE
catalog search path seems to not list all search paths

### ISSUE THREAD

**Issue by gwbischof** [bluesky/databroker#589]
![image](https://user-images.githubusercontent.com/11858848/91063197-ac63e000-e5fb-11ea-98d3-8724a10b672c.png)

**Comment by danielballan** [comment]
Yeah, I have bumped into this too. This currently shows at the search paths for v2-, intake-style config files. It does not show the search paths for v0-style config files. Needs thought about the consequences of returning a union of the two, mixing these different types.

**Comment by ronpandolfi** [comment]
I'm seeing this too, but with v2 style config files. `/usr/share/local/intake/*.yml` isn't listed by databroker, but is still discovered.
